.github/workflows/triage-pull-requests.yml

```diff
@@ -0,0 +1,21 @@
+name: Triage pull requests
+
+on:
+  pull_request:
+    types:
+      - labeled
+      - opened
+
+jobs:
+  add-to-project:
+    name: Triage pull requests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Add to project
+        uses: actions/add-to-project@v0.4.0
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+          # Add to 'Design System Sprint Board'
+          project-url: https://github.com/orgs/alphagov/projects/53
+          labeled: dependabot
```

Comment:
@domoscargin Are you happy with this only triaging Dependabot?

Reply:
For now, yeah. I'd expect anyone on the team to handle their own stuff.

"awaiting triage" tickets are kinda in the same boat of potentially getting missed, but we probably need a brief team discussion about how best to handle these - I feel like adding 'em to the backlog probably risks they get just as ignored as they often do.

Reply:
Don't think it excludes drafts but we can!

Ahh so quite often we just put an issue on the board, with related PRs only linked to the issue?

Reply:
Hmm, I personally like having all PRs in Needs Review with their description linking back to their Epic, but I'm not everyone else, so I'll pivot again (!) and say let's pop this in as just dependabot, and we can discuss other board management automation with the team.